Gemfile

```diff
@@ -38,7 +38,7 @@ group :development do
   gem 'quiet_assets'
   gem 'bullet'
   gem 'spring'
-  gem 'rubocop'
+#  gem 'rubocop'
```

Comment:
ho dovuto togliere rubocop perchè mi creava problemi in bundle install:
'Could not find ruby-progressbar-1.7.4 in any of the sources'